openforcefield/tests/test_molecule.py

```diff
@@ -688,6 +688,57 @@ def test_equality(self):
     # Test Molecule methods.
     # ----------------------
 
+    def test_find_rotatable_bonds(self):
+        """Test finding rotatable bonds while ignoring some groups"""
+
+        # test a simple molecule
+        ethanol = create_ethanol()
+        bonds = ethanol.find_rotatable_bonds()
+        assert len(bonds) == 2
+        for bond in bonds:
+            assert ethanol.atoms[bond.atom1_index].atomic_number != 1
+            assert ethanol.atoms[bond.atom2_index].atomic_number != 1
+
+        # now ignore the C-O bond
+        bonds = ethanol.find_rotatable_bonds('[#6:1]-[#8:2]')
+        assert len(bonds) == 1
+        assert ethanol.atoms[bonds[0].atom1_index].atomic_number == 6
+        assert ethanol.atoms[bonds[0].atom2_index].atomic_number == 6
```

Comment:
Could you copy this block and also run the check when the SMIRKS indices are reversed? The `sorted` line in the new function should handle it, but it's always good to be sure.

Reply:
Good point, I have added that as well.